WordPress/src/main/java/org/wordpress/android/util/extensions/DialogExtensions.kt

```diff
@@ -26,3 +31,22 @@ fun Dialog.setStatusBarAsSurfaceColor() {
         }
     }
 }
+
+fun BottomSheetDialog.fillScreen(isDraggable: Boolean = false) {
+    setOnShowListener {
+        val bottomSheet: FrameLayout = findViewById(
+            com.google.android.material.R.id.design_bottom_sheet
```

Comment:
❓ I was wondering if we should try to get this View without relying on the ID from the library 🤔 do you think that's possible/makes sense?